src/applications/simple-forms/form-upload/tests/helpers/index.unit.spec.js

```diff
@@ -1,37 +1,46 @@
 import sinon from 'sinon';
 import { expect } from 'chai';
 import * as actions from 'platform/forms-system/src/js/actions';
-import * as api from '@department-of-veterans-affairs/platform-utilities/api';
 import { shallow } from 'enzyme';
 import {
-  createPayload,
   getFileSize,
-  getFormUploadContent,
+  getFormNumber,
+  getFormContent,
   handleRouteChange,
   mask,
-  submitForm,
   uploadScannedForm,
 } from '../../helpers';
 
 describe('Helpers', () => {
-  describe('getFormUploadContent', () => {
-    it('returns the empty string when formNumber does not match', () => {
-      expect(getFormUploadContent('fake-form')).to.eq('');
+  describe('getFormNumber', () => {
+    it('returns correct path when formNumber matches', () => {
+      global.window.location = {
+        pathname: '/form-upload/21-0779/upload',
+      };
+      expect(getFormNumber()).to.eq('21-0779');
     });
-  });
 
-  describe('handleRouteChange', () => {
-    it('pushes the href to history', () => {
-      const fakeHref = 'fake-href';
-      const history = {
-        push(_) {},
+    it('returns empty string when formNumber does not match', () => {
+      global.window.location = {
+        pathname: '/form-upload/fake-form/upload',
       };
-      const historySpy = sinon.spy(history, 'push');
-      const route = { detail: { href: fakeHref } };
+      expect(getFormNumber()).to.eq('');
+    });
+  });
 
-      handleRouteChange(route, history);
+  describe('getFormContent', () => {
+    it('returns appropriate content when the form number is mapped', () => {
+      global.window.location = {
+        pathname: '/form-upload/21-0779/upload',
+      };
+      expect(getFormContent()).to.include({ title: 'Upload VA Form 21-0779' });
+    });
 
-      expect(historySpy.calledWith(fakeHref)).to.be.true;
+    it('returns default content when the form number is not mapped', () => {
```

Comment:
This can be a followup but I bet we don't actually want this to have a default. It should probably throw or redirect elsewhere or something else. But my understanding is that we're only turning form upload on for certain forms and won't want this kind of catch-all.

Reply:
Yeah, that makes sense. The line of reasoning here is that it's just a function that returns content. Any logic that would handle redirects or error throwing wouldn't happen here but further up the stack before this function would ever be called.